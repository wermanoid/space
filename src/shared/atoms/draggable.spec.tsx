import React from 'react';
import { render } from '@testing-library/react';

import Draggable from './draggable';

describe('sample', () => {
  it('should', () => {
    const { container } = render(
      <Draggable id="324" render="div">
        work
      </Draggable>
    );

    expect(container.firstChild).toMatchSnapshot();
    expect(container.firstChild).toHaveAttribute('draggable', 'true');
    expect(container.firstChild).toHaveTextContent('work');
  });
});
