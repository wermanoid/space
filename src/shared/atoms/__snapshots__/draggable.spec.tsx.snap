// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sample sjoudl 1`] = `
<Draggable
  id="324"
  render="div"
>
  <div
    draggable={true}
    onDragStart={[Function]}
  >
    work
  </div>
</Draggable>
`;
